config/routes.rb

```diff
@@ -113,6 +112,8 @@
   get 'basespace/projects', to: 'basespace#projects'
   get 'basespace/samples_for_project', to: 'basespace#samples_for_project'
 
+  get 'pipeline_viz', to: 'pipeline_viz#index'
```

Comment:
Probably matches a show route better:
`get '/pipeline_viz/:id', to: 'pipeline_viz#show'`

Reply:
Or even a nested resource: `/samples/:sample_id/pipeline_viz`?
Wdyt?

Reply:
Sounds good to me!